vm_supervisor/conf.py

```diff
@@ -221,5 +221,9 @@ class Config:
         env_file = ".env"
 
 
+def make_db_url():
+    return f"sqlite:///{settings.EXECUTION_DATABASE}"
```

Comment:
Would it make more sense to put this as a `@property` of the settings ?

```python
class Settings(BaseSettings):
    @property
    def execution_database_url(self):
        eturn f"sqlite:///{self.EXECUTION_DATABASE}"

Reply:
Doesn't make a lot of difference, maybe the advantage is that the caller does not have to get the singleton himself.